<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WizardPage</class>
 <widget class="QWizardPage" name="WizardPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>296</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <property name="title">
   <string>Arbeiter</string>
  </property>
  <property name="subTitle">
   <string>Erfassen der AKs</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_19">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Arbeiter</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="input_AK">
          <property name="editable">
           <bool>true</bool>
          </property>
          <property name="frame">
           <bool>true</bool>
          </property>
          <item>
           <property name="text">
            <string>Max</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Johannes</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>David</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_12">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Arbeitsbeginn</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="input_StartH">
          <item>
           <property name="text">
            <string>00</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>01</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>02</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>03</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>04</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>05</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>06</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>07</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>08</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>09</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>10</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>11</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>12</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>13</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>14</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>15</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>16</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>17</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>18</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>19</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>20</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>21</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>22</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>23</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>24</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_13">
          <property name="maximumSize">
           <size>
            <width>5</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:1, stop:0 rgba(32, 66, 19, 0), stop:1 rgba(255, 255, 255, 255));</string>
          </property>
          <property name="text">
           <string>:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="input_StartM">
          <item>
           <property name="text">
            <string>00</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>15</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>30</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>45</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_14">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Arbeitsende</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="input_EndeH">
          <item>
           <property name="text">
            <string>00</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>01</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>02</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>03</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>04</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>05</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>06</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>07</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>08</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>09</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>10</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>11</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>12</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>13</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>14</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>15</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>16</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>17</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>18</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>19</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>20</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>21</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>22</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>23</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>24</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_15">
          <property name="maximumSize">
           <size>
            <width>5</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:1, stop:0 rgba(32, 66, 19, 0), stop:1 rgba(255, 255, 255, 255));</string>
          </property>
          <property name="text">
           <string>:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="input_EndeM">
          <item>
           <property name="text">
            <string>00</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>15</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>30</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>45</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_16">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Pause</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="input_Pause">
          <property name="styleSheet">
           <string>background-color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_17">
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>Minuten</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_20">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Status</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="input_Status">
          <item>
           <property name="text">
            <string>Offen</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Ausgezahlt</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_5">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
